### PR TITLE
Add Nginx server and docker

### PR DESCRIPTION
**Pull request by bmath10**
The instructions for running nginx and docker are in the README.md, let me know if it doesn't work!

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:** @bmath10 
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/secure/RapidBoard.jspa?rapidView=55&projectKey=LANTERN&view=detail&selectedIssue=LANTERN-58
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [ ] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [ ] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [ ] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge is complete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,32 @@ npm install --save-dev @angular-devkit/build-angular
...
+### First Time
+1. Install nginx: `brew install nginx`
+2. Copy the local nginx config file `lantern.conf` to `/usr/local/etc/nginx/servers/`
+3. Make `lantern` directory in the nginx file system:
+    - `cd /usr/local/var/www`
+    - `mkdir lantern`
+
+### Every Time
+1. Build the production version of this project: `ng build --prod`
+2. Move the `dist` directory so nginx can run it: `mv ./dist/lantern-app/ /usr/local/var/www/lantern/`
```
Comment (mayerm94) [inline README.md]: I had to run `mv ./dist/lantern-app/* /usr/local/var/www/lantern/` to get this to work

File: README.md [hunk 2]
```diff
@@ -11,3 +11,32 @@ npm install --save-dev @angular-devkit/build-angular
...
+
+## Nginx server
+
+To run with the nginx server (on a Mac):
+
+### First Time
+1. Install nginx: `brew install nginx`
+2. Copy the local nginx config file `lantern.conf` to `/usr/local/etc/nginx/servers/`
+3. Make `lantern` directory in the nginx file system:
+    - `cd /usr/local/var/www`
```
Comment (mayerm94) [inline README.md]: Can be simplified to just `mkdir /usr/local/var/www/lantern`

File: README.md (hunk 1 above)
Comment (mayerm94) [reply README.md]: Also, `cp` might be better than 'mv` so that you don't have to rebuild to re-populate dist

File: README.md [hunk 3]
```diff
@@ -11,3 +11,24 @@ npm install --save-dev @angular-devkit/build-angular
...
+
+### Every Time
+1. Build the production version of this project: `ng build --prod`
+2. Move the `dist` directory so nginx can run it: `mv ./dist/lantern-app/ /usr/local/var/www/lantern/`
+3. Start the server: `sudo nginx -c /usr/local/etc/nginx/servers/lantern.conf`
+4. Go to `localhost:8090`
+
+### Stop the Server
+- To stop the server: `sudo nginx -c /usr/local/etc/nginx/servers/lantern.conf -s stop`
+- To stop nginx completely: `sudo nginx -s stop`
```
Comment (mrnosal) [inline README.md]: I'm getting errors using this to stop nginx: 
nginx: [emerg] "events" directive is not allowed here in /usr/local/etc/nginx/servers/lantern.conf:1

**Review by mrnosal — APPROVED** [review]
Fixes issues I had with starting and stopping nginx
